src/Microsoft.NET.Sdk.Functions.Generator/TypeUtility.cs

```diff
@@ -104,13 +104,22 @@ internal static Attribute GetResolvedAttribute(ParameterDefinition parameter, Cu
         public static Attribute ToReflection(this CustomAttribute customAttribute)
         {
             var attributeType = customAttribute.AttributeType.ToReflectionType();
+            if (attributeType == null)
+            {
+                throw new InvalidOperationException($"Could not load type '{customAttribute.AttributeType}'.");
+            }
 
             Type[] constructorParams = customAttribute.Constructor.Parameters
                  .Select(p => p.ParameterType.ToReflectionType())
                  .ToArray();
 
-            Attribute attribute = attributeType.GetConstructor(constructorParams)
-                .Invoke(customAttribute.ConstructorArguments.Select(p => NormalizeArg(p)).ToArray()) as Attribute;
+            ConstructorInfo ctor = attributeType.GetConstructor(constructorParams);
+            if (ctor == null)
+            {
+                throw new InvalidOperationException($"Can't find constructor for type {attributeType} matching params [{string.Join(",", constructorParams.Select(t => t.ToString()))}].");
```

Comment:
Minor comment -- can you change "Can't" to "Could not"?

Reply:
Done.